platform_versioned_docs/version-24.2/cli/commands.mdx

```diff
@@ -272,7 +272,176 @@ Run `tw compute-envs -h` to view the list of supported compute environment opera
   https://api.cloud.seqera.io/workspaces/97652229034xxx/datasets/6vBGj6aWWpBuLpGKjJDxxx/v/2/n/samplesheet_test.csv
   ```
 
-</details>  
+</details> 
+
+<details open>
+  <summary>**Studios**</summary>
+
+  Run `tw studios -h` to view the list of supported operations.
+
+  Manage [Studio sessions](../data_studios/index.mdx) hosted in Seqera Platform. Studio sessions allow interactive analysis using Jupyter, RStudio, VSCode and Xpra. Additional custom analysis environments can be defined as needed.
+
+  :::note
+  Most Studio operations require workspace `MAINTAIN` permissions. 
+  :::
+
+  ### List Studio sessions
+
+  Run `tw studios list -h` to view the required and optional fields for listing Studio sessions.
+
+  If a workspace is not defined, the `TOWER_WORKSPACE_ID` workspace is used by default.
+
+  ```bash
+  tw studios list -w community/showcase 
+
+  Studios at [community / showcase] workspace:
+
+     ID       | Name                            | Description                                        | User       | Status   
+    ----------+---------------------------------+----------------------------------------------------+------------+----------
+     13083356 | studio-jupyter-demo             | Demo Studio session for Jupyter                    | user1      | RUNNING  
+     419e602a | studio-rstudio-demo             | Demo Studio session for RStudio v4.4.1             | user2      | RUNNING  
+     722cb780 | studio-vscode-demo              | Demo Studio session for VSCode v1.83.0             | user3      | RUNNING  
+     950e56f5 | studio-multiqc-demo             | Demo MultiQC Studio session                        | user1      | RUNNING
+  ```
+
+  ### View a Studio session
+
+  Run `tw studios view -h` to view the required and optional fields for viewing session details.
+
+  ```bash
+  tw studios view -i 23ce7967 -w community/showcase
+
+    DataStudio at workspace '[community / showcase]'
+
+ ---------------------+------------------------------------------------------------
+  SessionID           | 23ce7967                                                   
+  Name                | experiment-analysis-session                                                     
+  Status              | STARTING                                                   
+  Status Last Update  | Fri, 31 Jan 2025 19:35:07 GMT                              
+  Studio URL          | https://a23ce7967.connect.cloud.seqera.io                  
+  Description         |                                                            
+  Created on          | Fri, 31 Jan 2025 18:12:27 GMT                              
+  Created by          | rob-newman | rob.newman@seqera.io                          
+  Template            | public.cr.seqera.io/platform/data-studio-jupyter:4.1.5-0.7 
```

Comment:
```suggestion
  Template            | public.cr.seqera.io/platform/studio-jupyter:4.1.5-0.7 
```

Reply:
Think this is moot. Its provided by the platform. Don't think we should change these yet.